src/components/list-address/AddressList.jsx

```diff
@@ -0,0 +1,39 @@
+import "./index.scss"
+
+const AddressList = ({ addresses, addAddress, removeAddress}) => {
+    const handleAddress = () => {
+        const address = prompt('Ingrese una nueva direccion ')
+        if (address){
+            addAddress(address)
```

Comment:
This would be renamed to : `onHandleAddress`, it's a better practice. Check concepts like uncontrolled and controlled components for a better comprehension.

Reply:
Rodrigo's component